save3ds_fuse/src/main.rs

```diff
@@ -856,7 +911,27 @@ where
         }
     }
 
-    fn opendir(&mut self, _req: &Request, ino: u64, _flags: u32, reply: ReplyOpen) {
+    fn release(
```

Comment:
It would be nice if you could preserve the original method order so the diff is smaller to make it easier to review